packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -58,22 +60,26 @@ export default class Classifier extends React.Component {
   render () {
     return (
       <Provider classifierStore={this.classifierStore}>
-        <ThemeProvider theme={{ mode: this.props.mode }}>
-          <Layout />
-        </ThemeProvider>
+        <Grommet theme={this.props.theme}>
+          <ThemeProvider theme={{ mode: this.props.mode }}>
+            <Layout />
+          </ThemeProvider>
+        </Grommet>
```

Comment:
If we're following the same model that the Classifier component is a consumable like the other components in react-components lib, then the responsibility of wrapping the it with Grommet to inject the theme is the consuming app. Adding this here creates a scenario where Grommet and the theme is added twice into the React context. Was this solving an issue or is there another reason to do this?

Reply:
Fixing an issue - I noticed that it didn't seem to be picking up the `lightGrey` colour properly.

Reply:
It doesn't seem like we should have to do this, but I'll let it go for this PR as it seems like that it might be a Grommet bug.